vmdb/app/models/ems_refresh/save_inventory.rb

```diff
@@ -207,7 +207,7 @@ def save_network_inventory(guest_device, hash)
   def save_networks_inventory(hardware, hashes, mode = :refresh)
     return if hashes.nil?
 
-    deletes = hardware.networks(true).dup
+    deletes = hardware.networks(true).to_a.dup
```

Comment:
There are a TON of these in the save_inventory*.rb files, but this PR doesn't seem to change them all...Should we?